docs/install.md

```diff
@@ -4,70 +4,88 @@ nav_order: 2
 
 # Trying out development builds
 
-## No default user accounts
+Before you build a [derived container image](https://gitlab.com/bootc-org/examples),
+you may want to just get a feel for the system, try out `bootc`, etc.  The bootable
+container images produced by this project are intended to be deployable in every
+physical and virtual environment that is supported by CentOS Stream 9 today.
 
-The default images produced do *not* include any default passwords or SSH keys.
-There is a `root` user present, but its password is locked.
+First, an important note to understand: the generic base container images
+do *not* include any default passwords or SSH keys.
 
-## Using the "generic cloud" image
+## Local virtualization (Linux & MacOS)
 
-Many people who just want to "try things out" will find it easiest to start
-with [the cloud image](https://github.com/CentOS/centos-bootc-layered/tree/main/cloud).
-(It's a separate container image because cloud-init does not work on every deployment
- target, and it also serves as an effective demonstration of layering)
+### podman desktop plugin (currently MacOS only)
 
-The [bootc playground](https://github.com/vrothberg/bootc-playground) repository
-helps automate this.
+There is a
+[podman desktop extension](https://github.com/containers/podman-desktop-extension-bootc)
+dedicated to this.
 
-## Use bootc-image-builder
+### podman-bootc-cli
 
-The [bootc-image-builder tool](https://github.com/osbuild/bootc-image-builder)
-supports generating disk images, including injecting user accounts.
+A new
+[podman-bootc-cli tool](https://gitlab.com/bootc-org/podman-bootc-cli) project offers
+a dedicated and streamlined CLI interface for running images, and in the future, it
+will become the backend for the podman desktop plugin.
 
-## Generating a raw disk image that can be launched via virt tooling
+### bootc-image-builder
 
-The above bootc-image-builder tool can generate disk images; however, a key part
-of the idea of `bootc` is that operating system images that use it are their
-own self-sufficient "baseline" installer.  So you can use this example:
+The [bootc-image-builder tool](https://github.com/osbuild/bootc-image-builder)
+supports generating a `qcow2` from our stock container image.
```

Comment:
Should state that it can produce AMI, RAW and ISO?

Reply:
I've tweaked both this and below to explain a bit more about what exists, but still not attempt to be exhaustive.